constrain/library/vav_turndown_using_average.py

```diff
@@ -0,0 +1,34 @@
+from constrain.checklib import RuleCheckBase
+
+
+class VAVTurndown_Using_Average(RuleCheckBase):
```

Comment:
Could @lymereJ think of a better name? Or is it ok as-is?

Reply:
Maybe `VAVTurndownDuringReheat`? Maybe #55 could be `VAVMinimumTurndownDuringReheat`?

Reply:
Sounds much better. Addressed.